### PR TITLE
[service.subtitles.ktuvit] 4.0.0

### PR DESCRIPTION
**Pull request by CaTzil**
### Description

[Ktuvit](https://ktuvit.me/) subtitle plugin suitable for Nexus version
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

### CODE REVIEW

File: service.subtitles.ktuvit/addon.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  <requires>
+    <import addon="xbmc.python" version="3.0.1"/>
+    <import addon="script.module.beautifulsoup4" version="4.8.2"/>
+  </requires>
+  <extension point="xbmc.subtitle.module" library="service.py"/>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Ktuvit.me</summary>
+    <description lang="en_GB">Search and Download subtitles from ktuvit.me.</description>
+    <source>https://github.com/XBMCil/service.subtitles.ktuvit</source>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
```
Comment (romanvm) [inline service.subtitles.ktuvit/addon.xml:12]: Please use SPDX license identifier: https://spdx.org/licenses/

File: service.subtitles.ktuvit/resources/lib/SUBUtilities.py [hunk 2]
```diff
@@ -0,0 +1,384 @@
+# -*- coding: utf-8 -*-
```
Comment (romanvm) [inline service.subtitles.ktuvit/resources/lib/SUBUtilities.py:1]: No need for this in Python 3

File: service.subtitles.ktuvit/service.py [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+    for file in xbmcvfs.listdir(__temp__)[1]:
+        full_path = os.path.join(__temp__, file)
+        if os.path.splitext(full_path)[1] in exts:
+            subtitle_list.append(full_path)
+
+    return subtitle_list
+
+
+def get_params(string=""):
```
Comment (romanvm) [inline service.subtitles.ktuvit/service.py:71]: This this ugly as hell function can be replaced with `dict(parse_qsl(sys.argv[2][1:]))`

**Review by romanvm — CHANGES_REQUESTED** [review]
Please fix `addon.xml`, other changes are not mandatory.

**Comment by CaTzil** [comment]
I've made the addon changes, can you review it?

**Comment by romanvm** [comment]
Please fix the license identifier and the addon will be ready to be merged. Other comments are not mandatory.

**Comment by CaTzil** [comment]
@romanvm, I've updated my PR, please review it 🙏🏽

**Comment by CaTzil** [comment]
Ping?

**Review by romanvm — APPROVED** [review]

**Pull request merged**